rfcs/formats/views/view-definition.md

```diff
@@ -19,53 +19,38 @@
 - What encoding do we use?
 - Encodings can be specifid by the creator (driver) and specified in `meta`.
 
-NOTE: initial proposal
-
 ```protobuf
-enum Protocol {
-  BITCOIN = 0;
-  ETHEREUM = 1;
-  FABRIC = 3;
-  CORDA = 4;
-}
-
-message View {
-  message Meta {
-    // Singleton, collection, computation
-    string type = 1;
-
+message Meta {
+    enum Protocol {
+        BITCOIN = 0;
+        ETHEREUM = 1;
+        FABRIC = 3;
+        CORDA = 4;
+    }
+    
     // Underlying distributed ledger protocol.
-    Protocol protocol = 2;
-
-    // What notion of time?
+    Protocol protocol = 1; 
+    
+    // What notion of time? 
     // If the observer and network are synchronizing on a global clock
     // there won't be a need to distinguish between static and dynamic views.
-    string timestamp = 3;
-
+    string timestamp = 2; 
+    
     // Notorization, SPV, ZKP, etc. Possibly enum
-    string proof_type = 4;
-
+    string proof_type = 3; 
+    
     // The data field's serialization format (e.g. JSON, XML, Protobuf)
-    string serialization_format = 5;
+    string serialization_format = 4;
+}
 
-    // External commitments made on this view
-    Commitment commitment = 6;
-  }
+message View {
   Meta meta = 1;
-
-  // Represents the data playload of this view.
+  
+  // Represents the data playload of this view. 
   // The representation of Fabric, Corda etc will be captured elsewhere.
   // For some protocols, like Bitcoin, the structure of an SPV proof is well known.
   bytes data = 2;
 }
-
-message Commitment {
-    // How is the commitment exchanged/located?
-    string address = 3;
-
-	// Cryptographic commitment scheme
-    string scheme = 1;
-}

```

Comment:
Can you leave this structure with a comment that we intend to add a commitment field to `Meta` in the future as a mechanism to make public commitments of state with freshness guarantees? (I don't want to lose this information.)